papers/whitepaper.md

```diff
@@ -111,3 +113,141 @@ The focus of the current text is to explore at a deeper level the Atoma's unique
+
+3. *Smart contract authenticated requests:* The second case, is the one in which the request is not started by a smart contract, but rather by a wallet, but it requires a smart contract to authenticate the request. Such examples, will include DAO governance decision making through AI, in which a transaction might be submitted through a multi-sig, but it requires approval directly from the majority of the DAO community, etc. Another interesting case is using account abstract 'wallet's to interact with the Atoma Network, in which one can leverage user-defined permission logic with decision making through AI (for example, an AI is required to parse the web to get some data information, and if that data satisfies some well defined logical criteria, blockchain transactions can be submitted). The latter case is especially interesting in the context of the autonomous Smart agents in the context of Web3 protocols. 
+Notice that, in such case the request input data needs not be on-chain data, or either public data and it can be submitted through a frontend UI, a desktop application, a mobile application, similar to the `Externally owned request` case. In this case, the transaction only goes through if the required smart contract 'authorizes' it through the Atoma on-chain logic contract. 
+
+Nodes on the Atoma Network listen to blockchain events that are emitted once new request transactions are submitted to the Atoma on-chain logic contract. Once it listens to a new event, it can check if it has been selected to process the request, or not. If it has been selected, it should start processing the request right away. If the event contains all the available input data (in case, the transaction was submitted another smart contract in a `Smart contract authenticated request` case), or it must then fetch the input data from an external API call (in case, the transaction was submitted through a `Externally owned request` case) to the application the user is interacted with, so that it can start processing the request, right away.
+
+**Node collateral and rewards:** As already mentioned, nodes on the Atoma Network are required to deposit a given amount of collateral, upon registration. This collateral is subject to slashing in the cases of malicious behavior or in case the node becomes unresponsive (it stops processing requests for a long period of time). It was already mentioned that nodes are only able to withdraw that collateral in the epoch following de-registration. 
```

Comment:
A previously mentioned nodes on the Atoma Network are required to deposit a given amount of collateral upon registration. This collateral is subject to slashing in the cases of malicious behavior or when the node becomes unresponsive (it stops processing requests for a long period of time). Note once more that nodes are only able to withdraw that collateral in the epoch following de-registration.